src/AppBundle/EventSubscriber/PullRequestSubscriber.php

```diff
@@ -41,13 +41,11 @@ public static function getSubscribedEvents(): array
                 ['checkForNewTranslations', 252],
                 ['initBranchLabel', 254],
                 ['initPullRequestTypeLabel', 254],
-                ['checkForClassicChanges', 252],
```

Comment:
Why not deleting the rest of the code?

Reply:
The rest of the code is able to parse diff and find whether the classic theme was modified. Could be useful for future, for example if we need a "Needs port to ClassicRocket theme". Basically I turned the feature flag off in this PR because I'm not sure the feature has to be deleted.